changelog/8178.bugfix.md

```diff
@@ -0,0 +1 @@
+Catch ChannelNotFoundEntity exception coming from the pika broker and raise as ConnectionException.
```

Comment:
@ancalita also worth using some markdown formatting for these in the future, for instance using backticks:
```
Catch `ChannelNotFoundEntity` exception coming from the pika broker and raise as `ConnectionException`.
```

That's probably nitpicking at this stage (sorry I was just passing by on your PR 😅 ), and I am not sure we even have guidelines for changelog fragments formatting 🤔 . I know we have [this page](https://www.notion.so/rasa/Docs-writing-guidelines-8c0ed021fb524f348bf94f9426439c86) documenting how we should write documentation pages though (worth taking a look at some point).